### PR TITLE
Added support for SRT and RIST streaming protocols

### PR DESCRIPTION
**Pull request by d-prieur**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-ffmpeg/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

Added support for the SRT protocol by pulling the latest tag from Haivision's GitHub repository and building it with shared libraries enabled using CMake & Make.
Added support for the RIST protocol by pulling the latest tag from VideoLAN's GitLab repository and building it with shared libraries enabled using Meson & Ninja.

I did not include the changes in the Dockerfile.aarch64 as I currently do not have any way of testing this architecture but to extend the functionality to it should probably be just a question of copying and pasting the code into the file.

## Benefits of this PR and context:

This extends the capabilities of the FFMPEG tool provided in this container image by allowing the user to stream in and out using the added modern video streaming protocols.

## How Has This Been Tested?

1. Checked that no build error occurred when creating the image from the Dockerfile
2. Checked that the protocols were available for both input and output with the "-protocols" command line option
3. Did a quick streaming test to show that FFMPEG in the container is able to stream a video using the protocols to FFPLAY on the host computer

## Source / References:

https://github.com/Haivision/srt
https://code.videolan.org/rist/librist

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -45,8 +45,10 @@ ENV \
   OPENJPEG=2.5.2 \
   OPUS=1.5.2 \
   RAV1E=0.7.1 \
+  RIST=0.2.10 \
   SHADERC=v2024.1 \
   SVTAV1=2.1.2 \
+  SRT=1.5.3 \
```
Comment (aptalca) [inline Dockerfile:51]: Should come before svtav1

File: Dockerfile [hunk 2]
```diff
@@ -858,6 +897,8 @@ RUN \
     --enable-libxml2 \
     --enable-libxvid \
     --enable-libzimg \
+    --enable-librist \
```
Comment (aptalca) [inline Dockerfile:900]: Please list in alphabetical order

**Review by aptalca — CHANGES_REQUESTED** [review]
Thanks for the PR.
Please add them to the aarch64 dockerfile as well

**Comment by d-prieur** [comment]
I have tried addressing all the requested changes. Please let me know if my updates are not what was expected or if there is anything I need to do to keep this moving along. Thank you.

**Review by aptalca — APPROVED** [review]

**Pull request merged**